ahab-maven-archetypes/ahab-simple-archetype/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.renci.ahab</groupId>
+        <artifactId>ahab-maven-archetypes</artifactId>
+        <version>0.1.7-SNAPSHOT</version>
```

Comment:
@hinchliff - 0.1.6? or snapshot?

Reply:
the versions in the poms will automatically get updated by Maven.

Reply:
Ah, yes, maven, this is what I get for reviewing Java code while having a Python project open in my IDE...